client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -486,6 +497,8 @@ export class _DisplayTable extends React.Component<
       .reverse()
       .value();
 
+    const footer_content = {};
```

Comment:
Was this supposed to be cleaned up? Doesn't seem used

Reply:
it was used in the "footer" of the table under the table closing tag. I just removed it since there's nothing we want in the "footer" right now